chain/client/src/metrics.rs

```diff
@@ -105,6 +105,24 @@ pub(crate) static VALIDATORS_CHUNKS_EXPECTED: Lazy<IntGaugeVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static VALIDATORS_CHUNKS_PRODUCED_BY_SHARD: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "near_validators_chunks_produced_by_shard",
+        "Number of chunks produced by a validator",
```

Comment:
Same description as the metric above.
Does this deprecate the other metric?

Reply:
From my searches online, you can do it. This means that we can deprecate the previous field. We will not do it in this PR because we need to figure out where it is used and migrate to the new metric.